ngraph/python/mock_py_frontend/mock_py_ngraph_frontend/mock_py_frontend.hpp

```diff
@@ -503,9 +503,10 @@ class MOCK_API FrontEndMockPy : public FrontEnd
 public:
     FrontEndMockPy(FrontEndCapFlags flags) { m_stat.m_load_flags = flags; }
 
-    InputModel::Ptr load_from_file(const std::string& path) const override
+    InputModel::Ptr load_impl(const std::vector<std::shared_ptr<Variant>>& params) const override
```

Comment:
Per my understanding, load_from_file shall be also overridden. Otherwise, call of 'load_from_file' will generate default "Not implemented" exception

Reply:
load_from_file was removed